corehq/apps/accounting/tasks.py

```diff
@@ -618,3 +620,90 @@ def assign_explicit_community_subscription(domain_name, start_date):
         internal_change=True,
         service_type=SubscriptionType.PRODUCT,
     )
+
+
+@periodic_task(run_every=crontab(minute=0, hour=9), queue='background_queue', acks_late=True)
+def send_overdue_reminders():
+    from corehq.apps.domain.views import DomainSubscriptionView
+    from corehq.apps.domain.views import DomainBillingStatementsView
+
+    today = datetime.date.today()
+    invoices = Invoice.objects.filter(is_hidden=False,
```

Comment:
super-nit: I see you're using different formatting in the two ```Invoice``` queries in this function (one using slashes between newlines, one without).  It can be clearer to stick to one approach, especially within the same function, unless you have a good reason not to.